### PR TITLE
`NetPolicyTimeoutsHostNetworkedPodTraffic`: Exclude cloud platforms from exposure

### PR DESCRIPTION
**Pull request by petr-muller**
- `NetPolicyTimeoutsHostNetworkedPodTraffic`: Add `_id==""` placeholders
- `NetPolicyTimeoutsHostNetworkedPodTraffic`: Limit exposure to baremetal-ish platforms

### CODE REVIEW

File: blocked-edges/4.12.48-NetPolicyTimeoutsHostNetworkedPodTraffic.yaml [hunk 1]
```diff
@@ -7,9 +7,17 @@ message: |-
   Clusters where Ingress Controller uses HostNetwork endpoint publishing strategy and certain NetworkPolicy configurations
   may experience network traffic disruptions in namespaces that allowed traffic before the upgrade.
 matchingRules:
-- type: PromQL
-  promql:
-    promql: |
-      group(max_over_time(apiserver_storage_objects{resource="networkpolicies.networking.k8s.io"}[1h]) > 0)
-      or
-      0 * group(max_over_time(apiserver_storage_objects[1h]))
+  - type: PromQL
```
Comment (petr-muller) [inline blocked-edges/4.12.48-NetPolicyTimeoutsHostNetworkedPodTraffic.yaml:10]: Apologize for the whitespace - it is coming from an automation which I'm not able to force to zero-indent :P

**Comment by petr-muller** [comment]
Tested on b03 cluster which does have networkpolicies:

```
$ b03 get networkpolicy --all-namespaces
NAMESPACE                                  NAME                                POD-SELECTOR                                                                              AGE
openshift-deployment-validation-operator   allow-from-openshift-insights       <none>                                                                                    465d
openshift-deployment-validation-operator   allow-from-openshift-olm            <none>                                                                                    465d
openshift-monitoring                       token-refresher                     app.kubernetes.io/component=authentication-proxy,app.kubernetes.io/name=token-refresher   717d
openshift-ocm-agent-operator               ocm-agent-allow-muo-communication   app=ocm-agent                                                                             25d
openshift-ocm-agent-operator               ocm-agent-allow-only-alertmanager   app=ocm-agent                                                                             323d
```

B03 is an AWS cluster so we do not want it to be exposed:

image

I do not have a non-cloud cluster at hand, so I will simulate by simply rewriting one of the |-ed items in the list of platforms to AWS to test the "platform matches and there are network policies" semantics:

image

Finally, test the same thing on b01 cluster that does not have any network policies, expect it to not match (even with modified list of platforms to include AWS):
```
$ b01 get networkpolicy --all-namespaces
No resources found
```
image

Test looks good to me.

File: blocked-edges/4.12.48-NetPolicyTimeoutsHostNetworkedPodTraffic.yaml [hunk 2]
```diff
@@ -7,9 +7,17 @@ message: |-
...
+    promql:
+      promql: |
```
Comment (sdodson) [inline blocked-edges/4.12.48-NetPolicyTimeoutsHostNetworkedPodTraffic.yaml:12]: Is this problematic?

Comment (sdodson) [reply blocked-edges/4.12.48-NetPolicyTimeoutsHostNetworkedPodTraffic.yaml:12]: I guess not given that's how it was before, nevermind.

**Review by wking — APPROVED** [review]
/lgtm

**Pull request merged**